src/dolphin/workflows/_cli_config.py

```diff
@@ -250,6 +260,31 @@ def get_parser(subparser=None, subcommand_name="run"):
         nargs=argparse.ZERO_OR_MORE,
         help="List the paths of all troposphere files to include.",
     )
+    correction_group.add_argument(
+        "--tropo-date-fmt",
+        default="%Y%m%d",
+        help="Format of dates contained in weather-model filenames.",
+    )
+    correction_group.add_argument(
+        "--tropo-package",
+        default="pyaps",
+        help="Package to use for tropospheric correction. Choices are: pyaps, raider.",
+    )
+    correction_group.add_argument(
+        "--tropo-model",
+        "--tropo-model",
+        default=TropoModel.ERA5.value,
+        choices=[t.value for t in TropoType],
```

Comment:
shoot sorry I messed up that last suggestion in 2 ways- there's the duped line, and i had `TropoType` not `TropoModel`

```suggestion
        default=TropoModel.ERA5.value,
        choices=[t.value for t in TropoModel],
```